microsoft-edge/extensions-chromium/publish/manage-settings.md

```diff
@@ -28,7 +28,11 @@ The **Contact info** section displays the details you provided when you created
   
 If your contact information changes edit the **Contact info** section.  Choose **Update**, make your changes, and then choose **Save**.  You may also choose **Contact info** in the left navigation menu to open the **Contact info** section.  
 
-If you need to change any of your contact details, the Microsoft Edge extension team may verify your account again, especially if you have a company account.  Your verification status returns to `Pending` while the review is in process.  The verification process ranges from a few days to a couple of weeks, and your company may receive a phone call from our verification partner.  
+If you need to change any of your contact details such as **Publisher display name (Company name)**, the Microsoft Edge extension team may verify your account again, especially if you have a company account.  Your verification status returns to `Pending` while the review is in process.  The verification process ranges from a few days to a couple of weeks, and your company may receive a phone call from our verification partner.
+To reflect the changes on the Edge add-ons though, you will need to submit an update, either the package update or metadata update, to your extension on Partner Center.
```

Comment:
Not sure what this is referring to? If this is about how to update your extension - dont we already have a document? If yes, we should just say - To update the add-ons published from your account refer to <link> documentation

Reply:
This is to call out this specifically "To reflect the changes on the Edge add-ons though, you will need to submit an update, either the package update or metadata update, to your extension on Partner Center."